### PR TITLE
feat: adding SES Receipt rule to receive FFIS emails and save them to S3

### PR DESCRIPTION
**Pull request by slapula**
## Description
Adding Terraform code for defining an SES receipt rule which receives FFIS emails to a dedicated Grants email address then saves that content to an S3 bucket.

### Automated and Unit Tests
- [ ] Added Unit tests

### Manual tests for Reviewer
- [ ] Added steps to test feature/functionality manually

## Checklist
- [ ] Provided ticket and description
- [ ] Provided testing information
- [ ] Provided adequate test coverage for all new code
- [ ] Added PR reviewers

### CODE REVIEW

File: terraform/main.tf [hunk 1]
```diff
@@ -204,6 +204,21 @@ data "aws_iam_policy_document" "read_datadog_api_key_secret" {
...
+  rule_set_name = "ffis_ingest-rule-set"
+  recipients    = [var.ffis_ingest_email_address]
+  enabled       = true
+  scan_enabled  = true
+  tls_policy    = "Require"
+
+  s3_action {
+    bucket_name       = module.grants_source_data_bucket.bucket_id
+    position          = 1
+    object_key_prefix = "/ses/ffis_ingest/new"
```
Comment (TylerHendrickson) [inline terraform/main.tf:218]: As we discussed in Slack, there's not a great way to test this outside of staging at the moment. However, dropping a note here just to keep it top-of-mind as something to check if we see issues – I've run into a few inconsistencies lately where S3 does (not) like the leading `/` at the beginning of object keys.

File: terraform/variables.tf [hunk 2]
```diff
@@ -182,3 +182,9 @@ variable "eventbridge_scheduler_enabled" {
   type        = bool
   default     = true
 }
+
+variable "ffis_ingest_email_address" {
+  type        = string
+  description = "Email address used to receive FFIS digests and save them to S3"
+  default     = "ffis-ingest@grants.usdigitalresponse.org"
+}
```
Comment (TylerHendrickson) [inline terraform/variables.tf:190]: Newline fix :P 

```suggestion
}

```

**Review by TylerHendrickson — APPROVED** [review]
💯

File: terraform/main.tf (hunk 1 above)
Comment (slapula) [reply terraform/main.tf:218]: Yeah, good catch. Let me change that before we roll this out.

**Pull request merged**